### PR TITLE
MODULAR ARCHITECTURE

### PR DESCRIPTION
**Pull request by Amedzro-Elikplim**
# AWESOME BOOKS ES6

- Implemented modular architecture
- Update scripts to type module
- Install Luxon to display the date in local time format
- Display date on all pages

### CODE REVIEW

File: modules/Date.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function showDate() {
+  const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
+  const div = document.querySelector('.date');
+  div.innerHTML = now;
+}
```
Comment (ShoiraTa) [inline modules/Date.js]: - [ ] Please make sure to use arrow syntax for functions. 
```
export const showDate = () => {
  const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
  const div = document.querySelector('.date');
  div.innerHTML = now;
}
```
And make sure to import it as  `import { showDate } from './Date.js';` in List.js and index.js files 😉

File: modules/Template.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  return li;
+}
+
+function li2(author) {
+  const li = document.createElement('li');
+  li.className = 'book-author';
+  li.innerHTML = author;
+
+  return li;
+}
```
Comment (ShoiraTa) [inline modules/Template.js]: - [ ] Same goes here ☀️, please make sure to use arrow syntax for functions

`const li1 = (title) => { logic }`

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @Amedzro-Elikplim ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my name

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Date.js (hunk 1 above)
Comment (Amedzro-Elikplim) [reply modules/Date.js]: Date.js file has a single function which is required to be the default export. eslint flags an error when a single function is not the default export. I, therefore, exported it as 
```
 const showDate = () => {
  const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
  const div = document.querySelector('.date');
  div.innerHTML = now;
} 

export default showDate;
```
and imported it as 
```
 import showDate from './Date.js';
```

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Amedzro-Elikplim, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Date.js (hunk 1 above)
Comment (ShoiraTa) [reply modules/Date.js]: Thanks for taking time to explain that, It is clear to me now why you decided to use function declaration 🙌🏻

**Pull request merged**